fluentlenium-core/src/main/java/org/fluentlenium/utils/chromium/ChromiumApi.java

```diff
@@ -0,0 +1,84 @@
+package org.fluentlenium.utils.chromium;
+
+import com.google.common.collect.ImmutableMap;
+import org.openqa.selenium.remote.Command;
+import org.openqa.selenium.remote.CommandExecutor;
+import org.openqa.selenium.remote.CommandInfo;
+import org.openqa.selenium.remote.HttpCommandExecutor;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.remote.SessionId;
+import org.openqa.selenium.remote.Response;
+import org.openqa.selenium.remote.http.HttpMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Method;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+import static org.fluentlenium.utils.chromium.Commands.SEND_COMMAND;
+import static org.fluentlenium.utils.chromium.Commands.SEND_COMMAND_AND_GET_RESULT;
+
+public class ChromiumApi {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ChromiumApi.class);
+    private final RemoteWebDriver remoteWebDriver;
+
+    public ChromiumApi(RemoteWebDriver remoteWebDriver) {
+        requireNonNull(remoteWebDriver, "WebDriver instance must not be null");
+        String browserName = remoteWebDriver.getCapabilities().getBrowserName();
+        if (!"chrome".equalsIgnoreCase(browserName)) {
```

Comment:
Good solution.

Should probably be !browserName.equalsIgnoreCase("chrome") though.

It shouldn't block merge but you may check latest Chromium-based Edge, Opera and Firefox (they started imlementing devtoolsprotocol).

Reply:
"chrome".equalsIgnoreCase(browserName) - generally it will avoid a null pointer exception.

It will be definitely worth to check on other browsers in near future.

Reply:
"chrome".equalsIgnoreCase(browserName) - generally this sequence will avoid a null pointer exception.

It will be definitely worth to check on other browsers in near future.

Reply:
"chrome".equalsIgnoreCase(browserName) - generally this syntax will avoid a null pointer exception.

It will be definitely worth to check on other browsers.